tests/Oryx.BuildImage.Tests/Hugo/HugoDynamicInstallationTest.cs

```diff
@@ -160,6 +160,43 @@ public void BuildsApplication_ByDynamicallyInstalling_IntoCustomDynamicInstallat
                 result.GetDebugInfo());
         }
 
+        [Fact]
+        public void BuildsApplication_ByDynamicallyInstallingIntoCustomDynamicInstallationDir()
+        {
+            // Arrange
+            var hugoVersion = "0.96.0";
```

Comment:
@samruddhikhandale I've approved it. But if you happen to have a chance parametrize the version for this test method that would be much appreciated 😄 

You could add a new `string` field to ImageData `public static TheoryData<string, string> ImageNameData` https://github.com/microsoft/Oryx/pull/1352/files#diff-2e7a27f5f7cd7f07d1391b0a49f7a29de67fee6e219e9e49f39c60bbaac3febeR30

Reply:
Example: https://github.com/microsoft/Oryx/pull/1349/files#diff-722368c372ac943fdf20a9ade036aa8cf25971b7581050d572e19bd49b47ba2bR28

Reply:
I'm working on a follow-up PR to address these comments. I want to merge this PR to avoid rebuilding of the pipeline for the change. I hope that's alright! 😄

Reply:
Thank you so much! Much appreciated :)